src/lib/utils.js

```diff
@@ -1,251 +1,253 @@
 export function formatNum(num, decimal) {
```

Comment:
Did anything in this file change other than formatting?

Reply:
@cKellyDesign sorry for the formatting inconsistencies my editor was set to js instead of javascript react. here is the change https://github.com/onaio/gisida-react/pull/173/files#diff-fb54f78b07de422efaa3d2d82bb82db3R230 the line is highlighted search for  level-view and
I introduced this prop to ensure the invisible fill layer that we needed to show the composite heatmaps would be moved to the top